src/main/java/org/fundacionjala/sfdc/core/driver/DriverManager.java

```diff
@@ -1,6 +1,7 @@
 package org.fundacionjala.sfdc.core.driver;
 
 
+import org.fundacionjala.sfdc.util.CommonActions;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.util.CommonActions'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)